Scenarios/Authentication/Mixed-auth-allow-anonymous-workaround/Service/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+
+var builder = WebApplication.CreateBuilder();
+
+builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
```

Comment:
It might be a good idea to add a note indicating that OpenIddict also has a validation handler that uses a more recent JWT stack, can easily switch to OAuth 2.0 introspection and is compatible with any standard OAuth 2.0 implementation (i.e not just `OpenIddict.Server`).